### PR TITLE
implement account api V3

### PR DESCRIPTION
**Pull request by oscarlopez8**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -4,4 +4,7 @@ browserName=chrome
 #Wait
 implicitWait=40
 explicitWait=30
-sleepTime=500
\ No newline at end of file
+sleepTime=500
```
Comment (LimbertVargas) [inline gradle.properties:7]: why do you implement sleep time?

File: gradle.properties [hunk 2]
```diff
@@ -4,4 +4,7 @@ browserName=chrome
...
+
+#Layout
+layoutpage=Lightning
```
Comment (LimbertVargas) [inline gradle.properties]: please, add an extra line.

File: src/main/java/salesforce/api/CaseAPI.java [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+     * @return the id of account created.
+     */
+    public String createAccount(final Map<String, String> newCase) {
+        finalEndpoint = CASE_ENDPOINT;
+        response = restClient.post(finalEndpoint, newCase);
+        return response.body().jsonPath().getString("id");
+    }
+
+    /**
+     * Deletes an Case by id.
```
Comment (LimbertVargas) [inline src/main/java/salesforce/api/CaseAPI.java]: please change "an" for "a"

File: src/main/java/salesforce/entities/Contact.java [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+ */
+public class Contact {
+
+    /**
+     * Id variable of Contact.
+     */
+    private String id;
+
+    /**
+     * Salutation Variablet of Contact.
```
Comment (LimbertVargas) [inline src/main/java/salesforce/entities/Contact.java]: fix "Variablet" for "variable"

File: src/main/java/salesforce/ui/pages/PageUserExperienceType.java [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+    }
+
+    /**
+     * Gets the name of page layout in which the tests are being executed.
+     *
+     * @return page layout.
+     */
+    public static String getPageLayoutName() {
+        properties = GetProperties.getProperties(PATH_OF_GRADLE_PROPERTIES_FILE);
+        //return = System.getProperty(LAYOUT).toUpperCase();
```
Comment (LimbertVargas) [inline src/main/java/salesforce/ui/pages/PageUserExperienceType.java:53]: please delete this comment line.

File: src/main/java/salesforce/ui/pages/home/HomePageClassic.java [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+
+    /**
+     * Opens the event form.
+     *
+     * @return an instance eventClassicForm.
+     */
+//    public EventFormAbstract openEventForm() {
+//        newEventBtn.click();
+//        return new EventClassicForm();
+//    }
```
Comment (LimbertVargas) [inline src/main/java/salesforce/ui/pages/home/HomePageClassic.java]: please delete this comment block

File: src/main/java/salesforce/ui/pages/home/HomePageLightning.java [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+    /**
+     * Waits until a webElement is loaded.
+     */
+    @Override
+    protected void waitUntilPageObjectIsLoaded() {
+
+    }
+
+    public void setSetupButton() {
```
Comment (LimbertVargas) [inline src/main/java/salesforce/ui/pages/home/HomePageLightning.java:46]: add documentation.

File: src/test/java/hook/AccountHook.java [hunk 8]
```diff
@@ -49,12 +53,26 @@ public AccountHook(final Context context) {
...
+     */
+    @Before("@create-account")
+    public void beforeScenario() {
+        List<Account> accounts = new ArrayList<>();
+        Map<String, String> createNewAccount;
+
+        for (Account ac : accounts) {
+//            createNewAccount = new HashMap<>();
+//            createNewAccount.put("Name", ac.getAccountName());
+//            account.setId(AccountAPI.getInstance().createAccount(createNewAccount));
```
Comment (LimbertVargas) [inline src/test/java/hook/AccountHook.java:74]: please delete this comment code

File: src/test/java/steps/LoginStep.java [hunk 9]
```diff
@@ -25,16 +24,23 @@
...
     /**
-     * This is the login Method
+     * This is the login Method.
      * @param userName parameter.
      * @param password parameter.
      */
-    @Given("I login with {string} and {string}")
-    public void iLoginWithAnd(final String userName, final String password) {
+    @Given("I am logged in SalesForce with {string} and {string}")
+    public void iAmLoggedInSalesForceWithAnd(final String userName, final String password) {
```
Comment (LimbertVargas) [inline src/test/java/steps/LoginStep.java:43]: change the name of the class accord Java conventions

**Review by LimbertVargas — COMMENTED** [review]
please review the obervations

File: gradle.properties (hunk 1 above)
Comment (oscarlopez8) [reply gradle.properties:7]: When initializes WebDriverManager.

File: src/main/java/salesforce/ui/pages/PageUserExperienceType.java (hunk 5 above)
Comment (oscarlopez8) [reply src/main/java/salesforce/ui/pages/PageUserExperienceType.java:53]: I tryin works to **System**.getProperty(LAYOUT).toUpperCase();
For this reason I need that line stay.

File: gradle.properties (hunk 1 above)
Comment (LimbertVargas) [reply gradle.properties:7]: ok

**Review by LimbertVargas — APPROVED** [review]

**Pull request merged**